app/V1Module/presenters/SubmitPresenter.php

```diff
@@ -366,15 +373,11 @@ public function actionResubmitAll(string $id) {
    * @param string $id identifier of assignment
    * @Param(type="post", name="files", validation="array", "Array of identifications of submitted files")
    * @throws NotFoundException
-   * @throws ForbiddenRequestException
    * @throws InvalidArgumentException
    * @throws ExerciseConfigException
    */
   public function actionPreSubmit(string $id) {
     $assignment = $this->assignments->findOrThrow($id);
-    if (!$this->assignmentAcl->canSubmit($assignment)) {
-      throw new ForbiddenRequestException("You cannot submit this assignment.");
-    }
 
```

Comment:
Why was the can submit check removed?

Reply:
As I said in description of PR, I strongly recommend to review this PR commit by commit.
But as for your question... Look UP ↑

Reply:
I see.